packages/forms/src/elements/MediaInput.tsx

```diff
@@ -14,20 +14,21 @@ import useFieldContext from '../utils/useFieldContext';
 import { VALIDATION } from '../utils/validation';
 
 export interface IMediaInputProps extends InputHTMLAttributes<HTMLInputElement> {
-  /** Apply compact styling */
+  /** Applies compact styling */
   isCompact?: boolean;
-  /** Remove borders and padding */
+  /** Removes borders and padding */
   isBare?: boolean;
-  /** Apply inset `box-shadow` styling on focus */
+  /** Applies inset `box-shadow` styling on focus */
   focusInset?: boolean;
-  /** Slot for "start" icon */
+  /** Accepts a "start" icon to display */
   start?: any;
-  /** Slot for "end" icon */
+  /** Accepts an "end" icon to display */
   end?: any;
+  /** Applies validation state styling */
   validation?: VALIDATION;
-  /** Apply props to the wrapping `FauxInput` element */
+  /** Applies props to the wrapping faux-input element */
   wrapperProps?: any;
-  /** Apply ref to the wrapping `FauxInput` element */
+  /** Applies a ref to the wrapping faux-input element */
```

Comment:
In these cases, it's best to keep `FauxInput`, as we're referring directly to the other component. Another option would (maybe preferable) would be to use:

```
[FauxInput](#fauxinput)
```

As a reference to the in-page API seen here: https://garden.zendesk.com/components/input#fauxinput

Reply:
Will do! I don't need the full link in there?

Reply:
No, we should optimize these for `website` usage. In fact, the [tweak](https://github.com/zendeskgarden/react-components/pull/954/commits/5b813ae810c31543b8da89cafc53dc99d2cd08af) I made to your `chrome` PR is currently resulting in an unwanted external-styled link. I'll need to go back and fix this.

<img width="745" alt="Screen Shot 2020-12-21 at 10 30 21 AM" src="https://user-images.githubusercontent.com/143773/102809849-8f57d780-4377-11eb-93a3-3777c0022fbd.png">

Reply:
So in this case it should just be `[brand color](#brand-colors)` ? I can add the fix if you would like (in this or a separate PR)

Reply:
It would have to be `[brand color](/design/color#brand-colors)` in order to be valid. A fix here would be great, thanks 🙏

Reply:
For clarification, the `MediaInput` lives on the same page as the `FauxInput`, so the link can be simplified to `[FauxInput](#fauxinput)`.